api/osrd_infra/serializers.py

```diff
@@ -34,13 +39,23 @@ class Meta:
         fields = "__all__"
 
 
+class RollingStockLiverySerializer(ModelSerializer):
+    class Meta:
+        model = RollingStockLivery
+        fields = ["id", "name"]
+
+
 class RollingStockSerializer(ModelSerializer):
+    liveries = RollingStockLiverySerializer(many=True, required=False)
```

Comment:
We should give an url instead of an ID. You can use https://www.django-rest-framework.org/api-guide/relations/#hyperlinkedrelatedfield to do so.